README.md

```diff
@@ -108,6 +112,108 @@ async onRequestInit({ requestInit }) {
+
+        // Read the request body
+        var json = await new StreamReader(HttpContext.Request.Body).ReadToEndAsync();
+
+        // Retrieve the Svix headers
+        var svixId = Request.Headers["svix-id"].ToString();
+        var svixTimestamp = Request.Headers["svix-timestamp"].ToString();
+        var svixSignature = Request.Headers["svix-signature"].ToString();
+
+        if (string.IsNullOrEmpty(svixId) || string.IsNullOrEmpty(svixTimestamp) || string.IsNullOrEmpty(svixSignature))
+        {
+            return BadRequest("Missing headers");
+        }
+
+        // Attempt signature verification with the raw signature
+        var wh = new Webhook(clerkWebhookSecret);
+        WebHeaderCollection headers = new WebHeaderCollection
+        {
+            { "svix-id", svixId },
+            { "svix-timestamp", svixTimestamp },
+            { "svix-signature", svixSignature }
+        };
+
+        Event webhookEvent;
+        try
+        {
+            wh.Verify(json, headers); // Verify doesn't return an event, just verifies the signature
+            webhookEvent = JsonSerializer.Deserialize<Event>(json); // Deserialize the JSON into an Event object
+        }
+        catch (Svix.Exceptions.WebhookVerificationException ex)
+        {
+            return BadRequest("Invalid signature");
+        }
+        catch (Exception ex)
+        {
+            return BadRequest("An error occurred");
+        }
+
+        switch (webhookEvent.Type)
+        {
+            case "user.created":
+                // Handle user created event
+                break;
+            case "user.updated":
+                // Handle user updated event
+                break;
+            case "user.deleted":
+                // Handle user deleted event
+                break;
+
+            default:
+                return BadRequest("Unhandled event type");
+        }
+
+        return Ok();
+    }
+
+    // Define the Event class to represent the webhook event
+    public class Event
+    {
+        public string? Type { get; set; }
+        public ClerkUser? Data { get; set; }
+    }
+
+    public class ClerkUser
+    {
+        public string? Id { get; set; }
+        public string? ExternalId { get; set; }
+
+        [JsonPropertyName("first_name")]
+        public string? FirstName { get; set; }
+
+        [JsonPropertyName("last_name")]
+        public string? LastName { get; set; }
+
+        [JsonPropertyName("email_addresses")]
+        public List<ClerkEmailAddress> EmailAddresses { get; set; } = new List<ClerkEmailAddress>();
+    }
+
+    public class ClerkEmailAddress
+    {
+        [JsonPropertyName("email_address")]
+        public string? EmailAddress { get; set; }
+    }
+```
+
 If the requests are coming from an SSR environment (ie NextJS), then you can either use the session JWT via the `__session` cookie and forward it on, or use a JWT template to create a unique JWT for your scenario.
```

Comment:
This text should be with the above section, looks like it got accidentally bumped.